freezer/freezer_url.go

```diff
@@ -44,28 +45,36 @@ func newFreezerSink(u *url.URL) (substrate.AsyncMessageSink, error) {
 	}
 
 	var streamstore straw.StreamStore
+	var err error
 	switch u.Scheme {
 	case "freezer+dir":
-		streamstore = &straw.OsStreamStore{}
+		streamstore, err = strawOpen("file:///")
```

Comment:
should it not open the given path?

Reply:
The way this is handled is that the stream store represents the root of the filesystem, and the path within is handled elsewhere at https://github.com/uw-labs/substrate/pull/83/files/88d34a3c2d8e1a84dad9bd8985e625e720356bb1#diff-13be8ef24263ac709ff59a9aa9b92fbfR88

Also, straw does not (yet) support the ability to obtain a stream store that is not a "root" of some kind. (root of fs, root of s3 bucket, etc)